.github/workflows/test.yaml

```diff
@@ -10,6 +10,10 @@ on:
         type: string
         description: The working directory for the docs
         default: "."
+      charmcraft-channel:
```

Comment:
There should be a place where charmcraft is installed. We should change that one too for consistency